property.go

```diff
@@ -13,7 +13,7 @@ type PropertyDefinition struct {
 	DisplayType           PropertyDefinitionDisplayTypeEnum `graphql:"displayType" json:"displayType"`
 	PropertyDisplayStatus PropertyDisplayStatusEnum         `graphql:"propertyDisplayStatus" json:"propertyDisplayStatus"`
 	LockedStatus          PropertyLockedStatusEnum          `graphql:"lockedStatus" json:"lockedStatus"`
-	Schema                JSON                              `json:"schema" scalar:"true"`
+	Schema                JSONSchema                        `json:"schema" scalar:"true"`
```

Comment:
Not sure why it was coded like this but in our API its actually this type (this struct isn't under codegen yet)